### PR TITLE
Add support for Bicep files

### LINKED ISSUE #4374: Add support for Bicep files
[Azure Bicep language](https://github.com/Azure/bicep) has a language server and [tree sitter grammar](https://github.com/Sjord/tree-sitter-bicep) that can be used to add support for it in Helix.

### PR DESCRIPTION
**Pull request by lunjon**
Closes #4374

### CODE REVIEW

File: runtime/queries/bicep/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+; Statements
+
+(object
+  (objectProperty
+    (identifier) @identifier))
+
+(propertyAccess
+  (identifier) @identifier)
+  
+(ifCondition) @conditional
```
Comment (the-mikedavis) [inline runtime/queries/bicep/highlights.scm:71]:
```suggestion
(ifCondition) @keyword.control.conditional
```

File: runtime/queries/bicep/highlights.scm [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  "existing"
+  "if"
+  "targetScope"
+  "output"
+] @keyword
+
+; Functions
+
+(decorator) @function.builtin
+(functionCall) @function.call
```
Comment (the-mikedavis) [inline runtime/queries/bicep/highlights.scm:18]:
```suggestion
(functionCall) @function
```

File: runtime/queries/bicep/highlights.scm (hunk 1 above)
Comment (lunjon) [reply runtime/queries/bicep/highlights.scm:71]: Thanks! Do you have a good resource/reference for available captures?

Comment (the-mikedavis) [reply runtime/queries/bicep/highlights.scm]: https://docs.helix-editor.com/master/themes.html#syntax-highlighting

**Review by the-mikedavis — APPROVED** [review]

**Comment by the-mikedavis** [comment]
Thanks!

**Pull request merged**